src/databricks/labs/ucx/installer/logs.py

```diff
@@ -191,10 +191,16 @@ def snapshot(self) -> list[LogRecord]:
             if 'databricks workspace export /' in message:
                 continue
             error_messages.append(message)
+            if "Couldn't fetch grants" in message and self._is_testing():
```

Comment:
@nkvuong : Could you add a comment explaining why we skip this in testing?

Reply:
added